bigchaindb/web/views/base.py

```diff
@@ -23,7 +23,7 @@ def make_error(status_code, message=None):
 
 def base_ws_uri():
     """Base websocket uri."""
-    scheme = config['wsserver']['scheme']
-    host = config['wsserver']['host']
-    port = config['wsserver']['port']
+    scheme = config['wsserver']['advertised_scheme']
```

Comment:
I didn't quite understand this. Use of the `advertised_*` variables are in the docs.
Do you mean replicate that over here?